lms/djangoapps/instructor/tests/test_services.py

```diff
@@ -114,3 +115,21 @@ def test_reset_non_existing_attempt(self):
             self.other_problem_urlname
         )
         self.assertIsNone(result)
+
+    def test_is_user_staff(self):
```

Comment:
I'm going to keep this because services.py exposes an in-proc API service. My calling site in edx-proctoring cannot import student.auth to use the methods directrly.